api_manager.py

```diff
@@ -2,20 +2,31 @@
 import tarot_api
 import animal_api
 import horoscope_api
+from fate_store import Fate
+from get_user_info import get_user_info
 
-card_meaning = tarot_api.tarot_meaning_request()   
-card_name = tarot_api.tarot_name_request()   
-animal_name = animal_api.animal_name_request()
-animal_picture = animal_api.animal_picture_request()
-users_horoscope = horoscope_api.horoscope_details()
-time = horoscope_api.time()
+zodiac_sign = get_user_info()[1]
+
+card_meaning = tarot_api.get_tarot_meaning()   
+card_name = tarot_api.get_tarot_name()   
+animal_name = animal_api.get_animal_name()
+animal_picture = animal_api.get_animal_picture()
+horoscope = horoscope_api.get_horoscope_details()
+time = horoscope_api.get_time_of_horoscope()
 
 api_data = {}
 
 
 def main():
 
     api_data.update({"Cards Name": card_name,"Tarot Card Meaning": card_meaning, "Animals Name": animal_name,
-    "Animal's Picture": animal_picture, "Time": time, "Users Horoscope": users_horoscope})
+    "Animal's Picture": animal_picture, "Time": time, "Users Horoscope": horoscope})
 
     return api_data
+
+def to_db():
+
+    animal = [animal_name, animal_picture]
+    tarot_card = [card_meaning, card_name]
+    return Fate(animal, tarot_card, horoscope, zodiac_sign)
+    
```

Comment:
Suggested refactor for api_manager: (this will cut down on a lot of superfluous code, and utilizes functions to avoid the issue of testing that comes up with global variables)

Line 8-15, the variable list, can be deleted.
Line 29 & 30 are also not needed.

`api_data = {}


def update_api_data():

    api_data.update({"Cards Name": tarot_api.get_tarot_name,"Tarot Card Meaning": tarot_api.get_tarot_meaning, "Animal's Name": animal_api.get_animal_name,
    "Animal's Picture": animal_api.get_animal_picture, "Time": horoscope_api.get_time_of_horoscope, "User's Horoscope": horoscope_api.get_horoscope_details})

    return api_data

def create_fate_object():

    return Fate(animal_api.get_animal, tarot_api.get_tarot_card, horoscope_api.get_horoscope, get_user_info()[1])`